src/Illuminate/Support/Fluent.php

```diff
@@ -46,6 +46,20 @@ public function get($key, $default = null)
     }
 
     /**
+     * Explicilty set an attribute.
+     *
+     * @param  string  $key
+     * @param  mixed   $value
+     * @return $this
+     */
+    public function set($key, $value = true)
```

Comment:
About `true` — it is the default in `__call`:
```php
     public function __call($method, $parameters)
     {
         $this->attributes[$method] = count($parameters) > 0 ? $parameters[0] : true;
```

I agree, a specific class for migrations would be wonderful, but this change is orthogonal to it (maybe it will help in other use cases of `Fluent`). Moreover, in that comment @taylorotwell said that he is not sure if he wants to go that way.